README.md

```diff
@@ -1,4 +1,14 @@
 # Clash Cores
-TODO
+**Warning: Under construction, expect force pushes to `main`**
+
```

Comment:
Is this true? I don't think we should force push anymore.

Reply:
*shrug* @lmbollen Thoughts? Not sure who the leader of this migration is.

Reply:
I wanted to have that there while we were still figuring things out with splitting off the repo and getting stuff working. I believe it can go now. Well, the _expect force pushes_ bit. It is still under construction. Mainly, I expect API changes before release.

Reply:
> Not sure who the leader of this migration is.

I think Lucas, yes? I'm also not sure. I mainly see Tim working on his PR's, but I am not aware of any progress on making `clash-cores` a functioning repository where PR's run and pass CI.